.pre-commit-config.yaml

```diff
@@ -28,6 +28,12 @@ repos:
       - id: isort
         args: [--settings-path, ./pyproject.toml]
 
+  - repo: https://github.com/pre-commit/mirrors-yapf
+    rev: v0.30.0
+    hooks:
+      - id: yapf
+        args: [--parallel]
```

Comment:
doesn't it also need -i and -r?

Reply:
yes, needs `-i`. Not `-r` because pre-commits applies each hook to the appropriate files already

```suggestion
        args: [--parallel, --in-place]
```

Reply:
The nice one is `-rip`